### PR TITLE
more physics removal for unused code

### PR DESCRIPTION
**Pull request by hannah-whelan**
https://github.com/isaaccomputerscience/isaac-cs-issues/issues/100

Tutor pages staying but not in use, Teacher Features removed, Isaac Card and Card Deck to be developed in future so left

Removed some lingering chemistry references

### CODE REVIEW

File: src/app/components/elements/modals/inequality/InequalityModal.tsx [hunk 1]
```diff
@@ -1,26 +1,14 @@
...
-} from "react";
+import React, { useCallback, useContext, useEffect, useLayoutEffect, useMemo, useRef, useState } from "react";
 import { Inequality, WidgetSpec } from "inequality";
 import { isDefined, parsePseudoSymbolicAvailableSymbols, sanitiseInequalityState } from "../../../../services";
 import { IsaacContentValueOrChildren } from "../../../content/IsaacContentValueOrChildren";
 import { ContentDTO } from "../../../../../IsaacApiTypes";
-import { Input } from "reactstrap";
 import classNames from "classnames";
 import { Markup } from "../../markup";
-import { CHEMICAL_ELEMENTS, EditorMode, LogicSyntax, MenuItemProps, MenuItems } from "./constants";
```
Comment (jmgq) [inline src/app/components/elements/modals/inequality/InequalityModal.tsx:18]: It looks like `CHEMICAL_ELEMENTS` is no longer used anywhere else, so it should also be removed from `constants.ts`.

File: src/app/components/content/IsaacCard.tsx [hunk 2]
```diff
@@ -5,6 +5,8 @@ import { apiHelper, isAppLink } from "../../services";
 import { Link } from "react-router-dom";
 import { IsaacCardDTO } from "../../../IsaacApiTypes";
 
+// NOTE: Currently not in use by CS. To be updated
```
Comment (jmgq) [inline src/app/components/content/IsaacCard.tsx:8]: I find these comments are a bit confusing. I think you're suggesting that there are plans to use this component in the future, but at the moment it's not used. If this is the case, then the comment should be updated with a reference to the ticket in which it's explained when and how this is going to be updated. And one of the items of the Acceptance Criteria in that ticket should be to remove these comments.

File: src/app/components/elements/modals/inequality/InequalityModal.tsx [hunk 3]
```diff
@@ -425,60 +396,10 @@ const InequalityMenu = React.forwardRef<HTMLDivElement, InequalityMenuProps>(
...
-                )}
-              </>
-            )}
           </div>
           <div id="inequality-menu-tabs" className="menu-tabs">
             <ul>
-              {["maths", "chemistry"].includes(editorMode) && (
-                <InequalityMenuTab menu={"numbers"} latexTitle={"1\\, 2\\, 3"} />
-              )}
+              {["maths"].includes(editorMode) && <InequalityMenuTab menu={"numbers"} latexTitle={"1\\, 2\\, 3"} />}
```
Comment (jmgq) [inline src/app/components/elements/modals/inequality/InequalityModal.tsx:402]: `["maths"].includes(editorMode)` can now be a simple comparison.

File: src/app/components/elements/modals/inequality/utils.ts [hunk 4]
```diff
@@ -1,6 +1,4 @@
 import {
-  CHEMICAL_ELEMENTS,
-  CHEMICAL_PARTICLES,
```
Comment (jmgq) [inline src/app/components/elements/modals/inequality/utils.ts:3]: `CHEMICAL_PARTICLES` can now be removed from `constants.ts`.

File: src/app/components/pages/Equality.tsx [hunk 5]
```diff
@@ -382,20 +381,6 @@ const Equality = withRouter(
               <Col md={{ size: 8, offset: 2 }} className="py-4 inequality-results">
                 <h4>LaTeX</h4>
                 <pre>${currentAttemptValue?.result?.tex}$</pre>
-                {editorMode === "chemistry" && (
-                  <>
-                    <h4>MhChem</h4>
-                    <pre>{currentAttemptValue?.result?.mhchem}</pre>
-                  </>
-                )}
-                {editorMode !== "chemistry" && (
```
Comment (jmgq) [inline src/app/components/pages/Equality.tsx:391]: Is this intended? The data inside this block will not show now, even if `editorMode` is `maths` or `logic`.

File: src/app/components/elements/modals/inequality/InequalityModal.tsx [hunk 6]
```diff
@@ -425,60 +396,10 @@ const InequalityMenu = React.forwardRef<HTMLDivElement, InequalityMenuProps>(
...
-              </>
-            )}
           </div>
           <div id="inequality-menu-tabs" className="menu-tabs">
             <ul>
-              {["maths", "chemistry"].includes(editorMode) && (
-                <InequalityMenuTab menu={"numbers"} latexTitle={"1\\, 2\\, 3"} />
-              )}
+              {editorMode === "maths" && <InequalityMenuTab menu={"numbers"} latexTitle={"1\\, 2\\, 3"} />}
               {["maths", "logic"].includes(editorMode) && (
```
Comment (jmgq) [inline src/app/components/elements/modals/inequality/InequalityModal.tsx:403]: This statement is now always true. I'm not sure if it's worth changing, so it's up to you.

Comment (hannah-whelan) [reply src/app/components/elements/modals/inequality/InequalityModal.tsx]: Makes sense for clarity, so I updated before merging it in

**Pull request merged**